amd64/root/firmware-update.sh

```diff
@@ -32,11 +35,13 @@ echo " "
 echo "Firmware available for flashing:"
 echo " "
 
-ls -1 /usr/share/deCONZ/firmware
+ls -1 "$FW_PATH"
 
 echo " "
-echo "Enter the firmware file name from above, including extension,"
-echo "or press Enter now to exit."
+echo "Enter the firmware file name from above, including extension."
+echo "Alternetivly you may enter the name of a firmware file to download"
+echo "from $FW_BASE"
+echo "Simply press Enter now to exit."
 echo " "
 
 read -p "File Name : " fileName
```

Comment:
A prefilling is usefull.
```suggestion
read -e -p "File Name : " -i \$FW_LATEST fileName
```

Reply:
But how do you make sure that the selection matches your hardware?